common/plugins/com.liferay.ide.core/plugin.xml

```diff
@@ -71,7 +71,16 @@
          point="org.eclipse.core.resources.natures">
       <runtime>
          <run
-               class="com.liferay.ide.core.LiferayNature">
+               class="com.liferay.ide.core.LiferayProjectNature">
+         </run>
+      </runtime>
+   </extension>
+   <extension
+         id="com.liferay.ide.core.liferayWorkspaceNature"
+         point="org.eclipse.core.resources.natures">
+      <runtime>
+         <run
+               class="com.liferay.ide.core.LiferayWorkspaceNature">
```

Comment:
Hey Greg, we need it for 
1) validation there is only one Liferay workspace project in the current eclipse workspace.
2) add the module projects to the liferay workspace project.
And some actions we need to know it is a "liferay workspace" project or not.

If we don't create a new nature for it, it is just a normal gradle project, right?